vaadin-grid-flow-parent/vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/vaadin-grid-flow-selection-column.html

```diff
@@ -3,12 +3,19 @@
 <dom-module id="vaadin-grid-flow-selection-column">
   <template>
     <template class="header" id="defaultHeaderTemplate">
+      <style>
+          /* Fix a grid web-component style that sets the font-size to small for all header contents */
+          #selectAllCheckbox {
```

Comment:
This is a new addition as it only applies to v14. It addresses a minor comment from https://github.com/vaadin/flow-components/issues/1060:

> Another minor flaw is that the checkbox in header appears to be smaller than checkboxes in column.